CMakeLists.txt

```diff
@@ -26,13 +26,16 @@ target_link_libraries(${PROJECT_NAME}_solver ${catkin_LIBRARIES} ${orocos_kdl_LI
 
 add_library(joint_state_listener src/joint_state_listener.cpp)
 target_link_libraries(joint_state_listener ${PROJECT_NAME}_solver ${orocos_kdl_LIBRARIES})
+target_compile_options(joint_state_listener PUBLIC -std=c++11)
```

Comment:
Instead of doing this, what if we just hoisted the

```
  if(NOT WIN32)
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
  endif()
```

from below and made it apply to the whole package?  I think this should be fine on all platforms that kinetic supports, right?